code/game/objects/items/constructionpermits.dm

```diff
@@ -0,0 +1,54 @@
+/* construction permits. Think blueprints but accessible to all engies and does NOT count as the antag steal objective
+these cannot rename rooms that are in by default BUT can rename rooms that are created via blueprints/permit*/
+
+/obj/item/blueprints/construction_permit
+	name = "construction permit"
+	desc = "An electronic permit designed to register a room for the use of APC and air alarms"
+	icon = 'icons/obj/items.dmi'
+	icon_state = "permit"
+	attack_verb = list("attacks", "baps", "hits")
+	w_class = W_CLASS_TINY
+
+
+	can_rename_areas = list(AREA_BLUEPRINTS)
+
+/obj/item/blueprints/construction_permit/attack_self
```

Comment:
I don't know either. This entire thing is mommi prints renamed with adjustments made to adapt it for use by carbons. I did test it and it does function as intended